src/disputables/queries.js

```diff
@@ -0,0 +1,17 @@
+import gql from 'graphql-tag'
+import { GraphQLWrapper } from '@aragon/connect-thegraph'
```

Comment:
I'm out of the loop on this one—is this better than the previous graphql transport we were using against the court subgraph?

Reply:
We could actually just do what the library does (creating the Client and performing the query) ourselves.  
It's using the same library internally.